attributes/default.rb

```diff
@@ -69,12 +69,21 @@
 case node['platform_family']
 
 when 'debian'
-  default['squid']['package'] = 'squid3'
-  default['squid']['config_dir'] = '/etc/squid3'
-  default['squid']['config_file'] = '/etc/squid3/squid.conf'
-  default['squid']['log_dir'] = '/var/log/squid3'
-  default['squid']['cache_dir'] = '/var/spool/squid3'
-  default['squid']['coredump_dir'] = '/var/spool/squid3'
+  if node['platform_version'] =~ /^8|^14/
```

Comment:
You'll want 9 in there as well

Reply:
Just checked in a kitchen bento box. It is named `squid` in debian 9. For some reason they changed from `squid` in debian 7 to `squid3` in debian 8 then back to `squid` in debian 9.

```bash
root@default-debian-9:~# dpkg-query -L squid | grep -E 'etc|var'
/etc
/etc/init.d
/etc/init.d/squid
/etc/logrotate.d
/etc/logrotate.d/squid
/etc/resolvconf
/etc/resolvconf/update-libc.d
/etc/resolvconf/update-libc.d/squid
/etc/squid
/etc/squid/errorpage.css
/etc/squid/squid.conf
/etc/ufw
/etc/ufw/applications.d
/etc/ufw/applications.d/squid
/var
/var/log
/var/log/squid
/var/run
/var/spool
/var/spool/squid
```